validator/server_arb/machine.go

```diff
@@ -300,6 +311,28 @@ func (m *ArbitratorMachine) DeserializeAndReplaceState(path string) error {
 	}
 }
 
+func (m *ArbitratorMachine) AddHotShotCommitment(height uint64, commitment []byte) error {
+	defer runtime.KeepAlive(m)
+	m.mutex.Lock()
+	defer m.mutex.Unlock()
+	if m.frozen {
+		return errors.New("machine frozen")
+	}
+
+	cbyte := CreateCByteArray(commitment)
+	status := C.arbitrator_add_hotshot_commitment(m.ptr, C.uint64_t(height), cbyte)
+	DestroyCByteArray(cbyte)
+	if status == 0 {
+		return nil
+	} else if status == 1 {
+		return errors.New("failed to add hotshot commitment: try into failed")
+	} else if status == 3 {
```

Comment:
I see - this answers my question above. For clarity, can we make these constants?